### PR TITLE
Pretix.PretixSCAN version 1.15.1

### PR DESCRIPTION
**Pull request by AlexMohanMD**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82621)

### CODE REVIEW

File: manifests/p/Pretix/PretixSCAN/1.15.1/Pretix.PretixSCAN.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Pretix.PretixSCAN
+PackageVersion: 1.15.1
+PackageLocale: en-US
+Publisher: rami.io GmbH
+PackageName: PretixSCAN
```
Comment (Trenly) [inline manifests/p/Pretix/PretixSCAN/1.15.1/Pretix.PretixSCAN.locale.en-US.yaml:8]:
```suggestion
PackageName: pretixSCAN
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by AlexanderMohan** [comment]
@cla accept

**Comment by AlexanderMohan** [comment]
@microsoft-github-policy-service agree [company="CHABGO Ltd."]

**Comment by AlexanderMohan** [comment]
@microsoft-github-policy-service agree company="CHABGO Ltd."

File: manifests/p/Pretix/PretixSCAN/1.15.1/Pretix.PretixSCAN.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Pretix.PretixSCAN
+PackageVersion: 1.15.1
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://marketplace.pretix.eu/download/pretixscan-windows/482
+  InstallerSha256: 863FAD54E18C9CF4F79125194B232DAD48D38764AA6F1E34600E9415B36C4EE1
```
Comment (ItzLevvie) [inline manifests/p/Pretix/PretixSCAN/1.15.1/Pretix.PretixSCAN.installer.yaml:10]:
```suggestion
  InstallerSha256: 863FAD54E18C9CF4F79125194B232DAD48D38764AA6F1E34600E9415B36C4EE1
  ProductCode: pretixSCAN
```
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\pretixSCAN`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 82821 to avoid pull request being stale;

**Pull request closed without merging**